artiq/coredevice/ad9910.py

```diff
@@ -65,6 +66,22 @@
 _DEFAULT_PROFILE_RAM = 0
 
 
+def AD9910_REG_FTW():
+    return _AD9910_REG_FTW
+
+
+def AD9910_REG_ASF():
+    return _AD9910_REG_ASF
+
+
+def AD9910_REG_PROFILE0():
+    return _AD9910_REG_PROFILE0
+
+
+def AD9910_REG_PROFILE7():
+    return _AD9910_REG_PROFILE7
+
+
```

Comment:
A bit weird and long. Just import the underscore names instead - Python allows it, and using them for the monitoring core is arguably internal use.